app/models/file_depot_ftp_anonymous.rb

```diff
@@ -1,26 +1,12 @@
 class FileDepotFtpAnonymous < FileDepotFtp
   def self.requires_credentials?
-    false
-  end
-
-  def remove_file(_file)
-    _log.info("Removing log file not supported on this depot type")
-  end
-
-  private
-
-  def create_directory_structure(_path)
-    nil
+    true
```

Comment:
@yrudman Why `true`?  This won't create any directory structure, and will probably blow up when trying to upload the file.